### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by ArielMAJ** [ArielMAJ/test.me#176] [Announcements]
## What's Changed
* [MOV][Issue #124] Created src folder. by @ArielMAJ in https://github.com/ArielMAJ/test.me/pull/136
* [ADD/UPT][Issue #134] Create files for core functions. by @tauanesales in https://github.com/ArielMAJ/test.me/pull/142
* [UPT][Issue #144] Update example image. by @tauanesales in https://github.com/ArielMAJ/test.me/pull/146
* [UPT][Issue #147] Update workflow. by @tauanesales in https://github.com/ArielMAJ/test.me/pull/148
* [UPT][Issue #143] Update how to contribute. by @tauanesales in https://github.com/ArielMAJ/test.me/pull/153
* [UPT][Issue #158] Update PT-BR translation. by @tauanesales in https://github.com/ArielMAJ/test.me/pull/160
* Add how to install valgrind and clang-format on CONTRIBUTING.md #159 by @PoulasthaMukherjee in https://github.com/ArielMAJ/test.me/pull/162
* [UPT][Issue #163] Added a commit message template to CONTRIBUTING.md.… by @ArielMAJ in https://github.com/ArielMAJ/test.me/pull/164
* [UPT][Issue #163] Update CONTRIBUTING.md by @tauanesales in https://github.com/ArielMAJ/test.me/pull/167
* [UPT][Issue #165] Add short video of the tool to README by @ArielMAJ in https://github.com/ArielMAJ/test.me/pull/168
* Improve argument check log in src/test.me.c by @tungbq in https://github.com/ArielMAJ/test.me/pull/172

### TLDR
* Updated project's folder structure;
* Update Makefile to make it more helpful;
* Update workflows to take into account above changes;
* Updated to README.md, CONTRIBUTING.md and PTBR translation.

## New Contributors
* @PoulasthaMukherjee made their first contribution in https://github.com/ArielMAJ/test.me/pull/162
* @tungbq made their first contribution in https://github.com/ArielMAJ/test.me/pull/172

Ty guys!

**Full Changelog**: https://github.com/ArielMAJ/test.me/compare/v1.1.2...v1.2.0
